dist/formats/get_involved/frontend/schema.json

```diff
@@ -304,6 +308,42 @@
         },
         "change_history": {
           "$ref": "#/definitions/change_history"
+        },
+        "take_part_pages": {
```

Comment:
Should these have been added to the `links` section, rather than `details`?

Reply:
Everything except for the .jsonnet is autogenerated as per the documentation, and the .jsonnet is correct based on other files, so this is just how messy these can become...

Reply:
Thanks for confirming!  This appears to be consistent with other links too.